cmd/minishift/cmd/delete.go

```diff
@@ -44,25 +44,31 @@ func runDelete(cmd *cobra.Command, args []string) {
 	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer api.Close()
 
+	removeConfigs()
```

Comment:
BTW, I was expecting that here in _delete.go_ the user interaction occurs. That we check whether the VM is registered and potentially print a message to the user